newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -165,6 +171,13 @@ export default class SearchPanel extends PureComponent<Props, State> {
                   this.setState({ searchInActions: checked })
                 }
               />
+              <InlineCheckbox
+                label={<Trans>Comments</Trans>}
```

Comment:
What do you think of "Texts" instead of "Comments"? Might be a bit more generic

Reply:
Yes more accurate